.github/workflows/pkg-holmes.yml

```
@@ -0,0 +1,30 @@
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Enable corepack (pnpm)
+        run: corepack enable
+      - name: Install deps
+        run: pnpm install --frozen-lockfile
+      - name: @wesley/holmes tests
+        run: pnpm --filter @wesley/holmes test
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)